ObjVisSAP.tex

```diff
@@ -385,26 +384,18 @@ \subsubsection{Non-compulsory Parameters}
 To be specified in Equatorial Coordinates J2000.
 </DESCRIPTION>
 </PARAM>
-<PARAM name="INPUT:T_MIN" ucd="time.start"
-utype="Char.TimeAxis.Coverage.Bounds.Limits.StartTime"
-datatype="double"
-unit="d" value="58171.45833">
-<DESCRIPTION>
-Specify the Start Time to check for visibility.
-To be specified in MJD.
-</DESCRIPTION>
-</PARAM>
-<PARAM name="INPUT:T_MAX" ucd="time.end"
-utype="Char.TimeAxis.Coverage.Bounds.Limits.StopTime" unit="d"
-datatype="double" value="58321.958333">
+<PARAM name="INPUT:TIME"
+utype="Char.TimeAxis.Coverage.Bounds.Limits"
+datatype="float"
+unit="d" value="58171.45833/58321.958333">
```

Comment:
Should this have `xtype="interval"`

Reply:
The ObjVisSAP protocol does not have any xtype defined for any of its fields. I think this is because it is more based on previous S*AP protocols where xtype was not used (SIAP 1.0, SSAP, SLAP, etc)
I think xtype appeared during the TAP discussions (to allow the description of ADQL types) and it was only incorporated in a S*AP for SIAP 2.0

Most probably, only this field is affected but I have raised a different issue
https://github.com/ivoa-std/ObjVisSAP/issues/13
for a more complete review

Reply:
Thanks, that sounds like a good approach.